randomness/src/jsligo/contracts/main.jsligo

```diff
@@ -65,7 +65,7 @@ let reveal = ([p, s] : [Parameter.Types.reveal_param, storage]) : [list<operatio
     };
     let new_locked : map<address, tez> = match (Map.find_opt(Tezos.sender, s.locked_tez), {
         None: () => failwith(Errors.wrong_user_balance),
-        Some: (val: tez) => Map.update(Tezos.sender, (Some(val - (10 as mutez))), s.locked_tez)
+        Some: (val: tez) => Map.update(Tezos.sender, (val - (10 as mutez)), s.locked_tez)
```

Comment:
Here the behaviour is different because if the negate fails (i.e return None) the value linked to the sender is removed. Is it the expected behaviour?

Reply:
fixed